process faults {6: ({1}, 3210561773)}
network faults {}
{5: ("C367DB57D7A5D1F45D8611D3B178A6BBEB887FEFFA7F7ED9D8B9107596425F97", {0, 5, 2, 3, 4, 1, 6}), 7: ("8550AFD103188A117518DD99CBD75B1900880460A5D740723EB42870CC94C9F2", {4, 5, 3, 1, 0, 6, 2}), 6: ("D7B339158481CA179E700BE31A3CA3A45F97FC73795FECF64581862F077FCB15", {1, 2, 4, 0, 5, 3, 6}), 8: ("91BB030B59A4408CA90E9B5218158E82D81604362D62431F07C734537F93DE25", {5, 3, 1, 6, 0, 2, 4}), 10: ("CE3F4A3379CD3FF0CF31DF02C77EC9323D571C620B18D3C6454E0239D9955A96", {1, 2, 5, 3, 6, 0, 4}), 9: ("FA1007B35B774265D18AF86913DD3654E085B707B2D97917D07FAD8F86FD5BE8", {4, 1, 6, 5, 2, 3, 0})}
7
reason: all committed
done!
